microprofile/messaging/src/test/java/io/helidon/microprofile/messaging/inner/subscriber/SubscriberPublToSubsBean.java

```diff
@@ -58,7 +59,7 @@
 
     @Override
     public void assertValid() {
-        assertTrue(resultData.containsAll(TEST_DATA));
-        assertEquals(TEST_DATA.size(), resultData.size());
+        assertThat(resultData.containsAll(TEST_DATA), is(true));
```

Comment:
There is an explicit assertion for this - 
`assertThat(resultData, Matchers.contains(TEST_DATA));` 
if TEST_DATA is an array or collection. Also if the order does not matter, there is a method containsInAnyOrder

Reply:
fixed